quickjs/src/native/Context.cpp

```diff
@@ -66,6 +66,7 @@ Context::Context(JNIEnv* env)
       doubleClass(static_cast<jclass>(env->NewGlobalRef(env->FindClass("java/lang/Double")))),
       objectClass(static_cast<jclass>(env->NewGlobalRef(env->FindClass("java/lang/Object")))),
       stringClass(static_cast<jclass>(env->NewGlobalRef(env->FindClass("java/lang/String")))),
+      memoryUsageClass(static_cast<jclass>(env->NewGlobalRef(env->FindClass("app/cash/quickjs/MemoryUsage")))),
```

Comment:
Is there a way to make this lookup fault tolerant? Like return null if type cannot be found?

That would allow us to use a more targeted shrinker rule and allow this class to be removed if unused.

Reply:
That’s a great idea. Lemme make a note of it!

Reply:
https://github.com/cashapp/quickjs-java/issues/255

Reply:
One really simple strategy for this – just defer looking up these types until we need ’em.